addon/bootstrap.js

```diff
@@ -7,8 +7,8 @@ function startup(data, reason) {
   });
 }
 
-function install() {
-}
+function shutdown(data, reason) {}
```

Comment:
Good question. That's not covered in the [MDN docs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Embedded_WebExtensions#Starting_the_WebExtension) for embedded webextensions, and to do so, we'd have to close over the `browser` object. I'm not sure if that's a good idea or not.

There are some bugs filed (#2332, #2333) around handling prefs to disable/enable the webextension, so maybe we can figure out the runtime.onMessage listener question there?